2018q4/riscv.md

```diff
@@ -0,0 +1,15 @@
+## FreeBSD/RISC-V update
+
+Contact: Ruslan Bukin <br@freebsd.org>
+
+Contact: Mark Johnston <markj@freebsd.org>
+
+FreeBSD/RISC-V is getting mature during last quarter.
+
+We have optimizied RISC-V copyin(9)/copyout(9) routines. They now supports word-sized copies where possible to dramatically increase speed of copying data between kernel and userspace.
+
+We made a series of improvements and bug fixes to pmap support (machine-dependent portion of virtual memory subsystem). This part was almost not touched during last years and now gets attention.
```

Comment:
I think this should be "not touched during the last years, and is now getting attention".